modules/typeform-webhook/modules/typeform-webhook/api/index.js

```diff
@@ -0,0 +1,76 @@
+import options from "../options";
+// @ts-ignore
+import { getGlobalOptions } from "@options";
+
+const global = getGlobalOptions();
+const BASE_URL = global;
+
+export const getForms = (oauthToken) => {
+  try {
+    const res = fetch("https://api.typeform.com/forms", {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${oauthToken}`
+      }
+    }
+    );
+    return res;
+  } catch (error) {
+    throw new Error("NETWORK_ERROR").message;
+  }
+};
+
+export const createWebHook = (oauthToken, formId, enabled) => {
+  try {
+    const res = fetch(`https://api.typeform.com/forms/${formId}/webhooks/cb-module`, {
```

Comment:
`cb-module` this part of the URL is it something thats going to change with each project? then maybe have it in options or a separate variable and document it.

Reply:
Moved it to options.